modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/css/_balloontoolbar.scss

```diff
@@ -0,0 +1,53 @@
+.lfr-balloon-editor {
+	&.cke_balloon.cke_balloontoolbar {
+		background: white;
+		border-color: #e7e7ed;
+		box-shadow: 0 8px 16px rgba(0, 0, 0, 0.2);
+		.cke_balloon_triangle_outer.cke_balloon_triangle_bottom {
+			border-color: #e7e7ed transparent;
+		}
+		.cke_balloon_triangle_outer.cke_balloon_triangle_top {
+			border-color: #e7e7ed transparent;
+		}
+		.cke_balloon_triangle_inner.cke_balloon_triangle_bottom {
+			border-color: white transparent;
+		}
+		.cke_balloon_triangle_inner.cke_balloon_triangle_top {
+			border-color: white transparent;
+		}
+		span {
+			&:last-child {
```

Comment:
Minor SF: if there is only one child, you can collapse. In this case, to `span:last-child`

Reply:
Makes sense! Changing!